Makefile

```diff
@@ -19,13 +22,19 @@ help:
 	@echo '    build           Build package'
 	@echo '    clean           Clean directories'
 
-.PHONY: setup
-setup:
+venv:
+	test -d $@ || mkdir -m 755 ./$@
+
+$(VENV_NAME): venv
 	test -d $(VENV_NAME) || python -m venv $(VENV_NAME)
+	test -d $(VENV_NAME) || exit 1 ; # venv not working right
```

Comment:
Should this be obvious from error messages when setting it up?

Reply:
Example case: macOS developer tools edition of `make` will for example, and I've seen this in the field on many occasions antidotally, and while my environment may be niche, I need this to get it working in my environment.

Reply:
Thanks for the clarification. Please either update the comment and ensure that it is preceded by two whitespace characters or remove it. As it stands, it does not test if the virtual environment is working right, but only if the corresponding directory exists.